<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizPLCLiveView</class>
 <widget class="QWidget" name="WizPLCLiveView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTest1">
       <property name="text">
        <string>Enqueu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnTest2">
       <property name="text">
        <string>Dequeue</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="WizMeterView" name="meterView11" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView10" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView9" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView8" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView7" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView6" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView5" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView4" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView3" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView2" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView1" native="true"/>
     </item>
     <item>
      <widget class="WizMeterView" name="meterView0" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WizMeterView</class>
   <extends>QWidget</extends>
   <header location="global">wizmeterview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
